core/poi/lib/utils/validateConfig.js

```diff
@@ -84,7 +85,8 @@ module.exports = (api, config) => {
           struct([struct.union(['object', 'function'])])
         ])
       ),
-      open: 'boolean',
+      open: struct.union(['boolean', 'string']),
```

Comment:
we don't have `open: string` support in Poi. that's a webpack-cli / webpack-dev-server cli feature

Reply:
Ah okay, is it safe to leave the history api fallback as an allowed option? I'll just revert the second commit otherwise

Reply:
that's ok, I just updated the docs https://poi.js.org/config.html#devserver-historyapifallback